updater/reports/ReportRepoUsage.py

```diff
@@ -0,0 +1,53 @@
+from .ReportDaily import *
+
+# Calculate percentage of active repositories over the last four weeks
+# in organizations that have received pushes to at least one protected
+# branch and that have at least one topic assigned to them.
+class ReportRepoUsage(ReportDaily):
+	def name(self):
+		return "repository-usage"
+
+	def updateDailyData(self):
+		newHeader, newData = self.parseData(self.executeQuery(self.query()))
+		self.header = newHeader if newHeader else self.header
+		self.data.extend(newData)
+		self.truncateData(self.timeRangeTotal())
+		self.sortDataByDate()
+
+	def query(self):
+		oneDayAgo = self.yesterday()
+		fourWeeksAgo = self.daysAgo(28)
+		query = '''
+			SELECT
+				"''' + str(oneDayAgo) + '''" AS date,
+				ROUND(100*COUNT(DISTINCT(protect.repository_id))/COUNT(DISTINCT(active.repository_id)))
```

Comment:
Can you put spaces around the arithmetic operators for consistency?

Reply:
Also, isn’t a float conversion necessary here to avoid an integer division?

Reply:
For the record, MySQL seems to perform a float division with `/` in all cases, while integer divisions require the `DIV` keyword. That’s strange, because other SQL implementations handle this differently as it seems …